front/src/applications/operationalStudies/views/Scenario/components/ManageTimetableItem/Itinerary/PathStepItem.tsx

```diff
@@ -39,6 +39,39 @@ const PathStepItem = ({
   const mapSettings = useMapSettings();
   const { updateViewport } = useMapSettingsActions();
 
+  const getInvalidMessage = () => {
+    let message = t('invalidOP');
+    if (!pathStepMetadata?.isInvalid || !pathStep?.location) return message;
+
+    const { location } = pathStep;
+
+    if ('track' in location) {
+      return (message += `${t('requestedPoint')}`);
```

Comment:
```suggestion
      return `${message}${t('requestedPoint')}`;
```

why not use full interpolation ?